docs/external plugins/Verifier/vulnerabilityreport.md

```diff
@@ -0,0 +1,327 @@
+  parameters:
+    maximumAge: 24h
+    disallowedSeverity:
+      - high
+      - critical
+    denylistCVEs:
+      - CVE-2021-44228 # Log4Shell
+```
+
+| Name               | Required | Path                               | Description                                                                                            | Default Value |
+| ------------------ | -------- | ---------------------------------- | ------------------------------------------------------------------------------------------------------ | ------------- |
+| maximumAge         | No       | spec.parameters.maximumAge         | The string formatted max age of report                                                                 | ""            |
+| disallowedSeverity | No       | spec.parameters.disallowedSeverity | String array of disallowed severities. Verification fails if ANY specified severity found              | []            |
+| denylistCVEs       | No       | spec.parameters.denylistCVEs       | String array of CVE IDs. Verification fails if ANY specified CVE ID found                              | []            |
+| passthrough        | No       | spec.parameters.passthrough        | Bypasses all verification except for `maximumAge`. Report content in `report` field of verifier report | false |
+
+### CLI
+
+Sample JSON
+```json
+{
+    "store": {
+        "version": "1.0.0",
+        "plugins": [
+            {
+                "name": "oras",
+            }
+        ]
+    },
+    "policy": {
+        "version": "1.0.0",
+        "plugin": {
+            "name": "configPolicy"
+        }
+    },
+    "verifier": {
+        "version": "1.0.0",
+        "plugins": [
+            {
+                "name": "vulnerabilityreport",
+                "artifactTypes": "application/sarif+json",
+                "maximumAge": "24h",
+                "disallowedSeverity": ["high", "critical"],
+                "denylistCVEs": ["CVE-2021-44228"]
+            }
+        ]
+    }
+}
+```
+
+## Advanced Scenario
+
+Alice has a Kubernetes cluster. She wants to enable vulnerability report verification. In particular, she wants to make sure all container images used in K8s resources have at least one valid vulnerability report attached to the image. The report must:
+- be in SARIF format
+- generated from Trivy or Grype
+- created less than 24 hours ago
+- has no HIGH or CRITICAL severity level vulnerabilities in the report
+- does not contain the log4shell CVE 
+
+Furthermore, each vulnerability report must have a valid notation signature attached to it.
+
+First, follow only steps 1 and 2 of the [manual quickstart](../../quickstarts/quickstart-manual.md). 
+
+Next, install the vulnerability report constraint template and constraint:
+
+```shell
+kubectl apply -f https://deislabs.github.io/ratify/library/vulnerability-report-validation/template.yaml
+
+kubectl apply -f https://deislabs.github.io/ratify/library/vulnerability-report-validation/samples/constraint.yaml
```

Comment:
For advanced scenario, do you want to elaborate more on how users apply the flexible constraint policies based on the given condition " all container images used in K8s resources have at least one valid vulnerability report attached to the image."?

Reply:
Added a bit more context on the exact policy constraint template defines